lldb/CMakeLists.txt

```diff
@@ -38,6 +38,10 @@ endif()
 include(LLDBConfig)
 include(AddLLDB)
 
+if (UNIX AND ${CMAKE_SYSTEM_NAME} MATCHES "AIX")
+  add_definitions("-D__AIX__")
```

Comment:
This looks like a macro that a compiler might set for you, is that not the case here?

Reply:
Also remember that users may be debugging AIX from a non-AIX host where the compiler would not be setting any macros like that.

(though perhaps this is exactly why you defined this manually, just to get it working on AIX only as the first step)